libpod/boltdb_state.go

```diff
@@ -1101,6 +1107,382 @@ func (s *BoltState) PodContainers(pod *Pod) ([]*Container, error) {
 	return ctrs, nil
 }
 
+// AddVolume adds the given volume to the state. It allso adds ctrDepID to
+// the sub bucket holding the container dependencies that this volume has
+func (s *BoltState) AddVolume(volume *Volume, ctrDepID string) error {
```

Comment:
Is it OK to create a volume with no container dependencies?

I think you might be better off having AddVolume just make the volume, and then using UpdateVolume for adding/removing all the dependency IDs

Reply:
Yup, it is. Just made it happen only in UpdateVolume.